lib/src/kvs/ds.rs

```diff
@@ -529,44 +533,65 @@ impl Datastore {
 		Ok(())
 	}
 
-	pub async fn nuke_whole_cluster(&self, tx: &mut Transaction) -> Result<(), Error> {
+	pub async fn clear_unreachable_state(&self, tx: &mut Transaction) -> Result<(), Error> {
 		// Scan nodes
-		let cls = tx.scan_cl(1000).await?;
-		trace!("Found {} nodes", cls.len());
-		for cl in cls {
-			tx.del_cl(
+		let cluster = tx.scan_nd(1000).await?;
+		trace!("Found {} nodes", cluster.len());
```

Comment:
not perse this scan, but did we also want to loop until no more resources were found in some spots?

Reply:
I have removed limits for nd and hb scans when limit is set to 0. Overall, this is bad practice, since hypothetically it could be in the gazillions. But realistically, it won't. And if it is, then we could patch it. Thanks for raising 🙌